components/das_disconnect_message.py

```diff
@@ -0,0 +1,33 @@
+from components import Message
+from canvas import Canvas, Colour
+from data import Data
+
+
+class DASDisconnectMessage(Message):
+    """Class displays error message on overlay when DAS is not connected.
+
+    Message can only be received when the DAS is not connected. Will
```

Comment:
`received` -> `displayed`?

Reply:
Actually on second thought we don't need this sentence since it conveys the same message as the one previous to it.

Reply:
Hm, I was thinking about this, but I wanted this to be clearly pointed out in the docstring, so that there's no misunderstanding in the future that the disconnected message would appear at a random different time **other** than when the DAS is disconnected. 

Maybe I could change the first line of the docstring to "Class will display an error message on the overlay that the DAS is not connected."